src/parser/jobs/blm/index.js

```diff
@@ -16,12 +15,6 @@ export default new Meta({
 
 	Description: () => <>
 		<TransMarkdown source={description}/>
-		<Message warning icon>
-			<Icon name="warning sign"/>
-			<Message.Content>
-				<Trans id="blm.about.description.warning">This isn&apos;t even remotely done.</Trans>
-			</Message.Content>
-		</Message>
```

Comment:
Since you removed this, can you run `yarn extract` again?

Reply:
oh yea, sorry